tools/flang1/flang1exe/semgnr.c

```diff
@@ -941,8 +943,9 @@ find_operator(int opr, SST *lop, SST *rop, LOGICAL elemental)
     int sptrgen = sptr;
     if (NMPTRG(sptrgen) != nmptr)
       continue;
-    if (STYPEG(sptrgen) == ST_ALIAS)
+    if (STYPEG(sptrgen) == ST_ALIAS) {
```

Comment:
again, what is the effect of this change?

Reply:
This is also just a formatting style change.